src/net/sourceforge/kolmafia/textui/command/SendMessageCommand.java

```diff
@@ -65,17 +65,29 @@ public static void send(final String parameters, final boolean isConvertible) {
         ItemFinder.getMatchingItemList(itemList, KoLConstants.inventory);
 
     if (attachments.length == 0
-        && (itemList.length() > 1 || message == KoLConstants.DEFAULT_KMAIL)) {
+        && (itemList.length() > 1 || message.equals(KoLConstants.DEFAULT_KMAIL))) {
       return;
     }
+    /*
```

Comment:
This is the code where the rewrite has different coverage than the original and I can't figure out why.